lib/bmg/operator/minus.rb

```diff
@@ -0,0 +1,43 @@
+module Bmg
+  module Operator
+    #
+    # Minus operator.
+    #
+    # Returns all tuples which are in the left operand but not 
+    # in the right operand.
+    #
+    # This implementation is actually a NAry-Minus, since it handles
+    # an arbitrary number of operands.
+    #
+    class Minus
+      include Operator::Nary
+
+      def initialize(type, operands)
+        @type = type
+        @operands = operands
+      end
+
+    public
+
+      def all?
+        false 
+      end
```

Comment:
why all?

Reply:
was under the impression some part of translation of nary operations expected it, but you're right it's not needed, will remove.